From 342abf5122d9a8f007c8804906fb3e2a5efc7579 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Sun, 4 Oct 2026 09:58:08 -0400
Subject: [PATCH] Refresh the guest package lock

ffmpeg 2:9.0.2-1 -> 2:9.0.2-2
glslang 1:1.4.357.0-1 -> 1:1.4.363.0-1
hyprland 0.56.2-3 -> 0.56.2-4
leancrypto 1.9.0-1 -> 1.9.1-1
libmd 1.2.0-1 -> 1.3.0-1
libplacebo 7.360.1-6 -> 7.360.1-7
libutf8proc 2.11.3-1 -> 2.12.0-1
md4c 0.5.3-1 -> 0.6.0-2
mesa 1:26.2.3-2 -> 1:26.2.4-1
mkinitcpio 42.1-1 -> 42.2-1
openexr 3.5.1-1 -> 3.5.2-1
sdl2-compat 2.32.72-1 -> 2.32.74-1
sdl3 3.4.16-1 -> 3.4.18-1
shaderc 2026.3-1 -> 2026.4-1
spirv-tools 1:1.4.357.0-1 -> 1:1.4.363.0-1
vulkan-icd-loader 1.4.357.0-1 -> 1.4.363.0-1
vulkan-mesa-implicit-layers 1:26.2.3-2 -> 1:26.2.4-1
vulkan-swrast 1:26.2.3-2 -> 1:26.2.4-1
vulkan-virtio 1:26.2.3-2 -> 1:26.2.4-1
---
 guest/packages.lock.json | 38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index df5fc84..2b95f75 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -77,7 +77,7 @@
     "fcitx5-gtk": "5.1.7-1",
     "fcitx5-qt": "5.1.16-1",
     "fd": "10.5.0-3",
-    "ffmpeg": "2:9.0.2-1",
+    "ffmpeg": "2:9.0.2-2",
     "fftw": "3.3.11-1",
     "file": "5.48-1",
     "filesystem": "2025.10.12-1",
@@ -109,7 +109,7 @@
     "glib-networking": "1:2.80.1-1",
     "glib2": "2.88.3-1",
     "glibc": "2.44+r50+g1848099f063e-1",
-    "glslang": "1:1.4.357.0-1",
+    "glslang": "1:1.4.363.0-1",
     "glu": "9.0.3-3",
     "glycin": "2.1.5-2",
     "glycin-gtk4": "2.1.5-2",
@@ -162,7 +162,7 @@
     "hyprcursor": "0.1.13-7",
     "hyprgraphics": "0.5.1-4",
     "hypridle": "0.1.8-2",
-    "hyprland": "0.56.2-3",
+    "hyprland": "0.56.2-4",
     "hyprland-guiutils": "0.2.2-4",
     "hyprland-preview-share-picker": "0.2.1-1",
     "hyprlang": "0.6.8-5",
@@ -196,7 +196,7 @@
     "l-smash": "2.14.5-4",
     "lame": "4.0-1",
     "lcms2": "2.19.1-1",
-    "leancrypto": "1.9.0-1",
+    "leancrypto": "1.9.1-1",
     "leptonica": "1.87.0-2",
     "less": "1:710-1",
     "libadwaita": "1:1.9.4-1",
@@ -292,7 +292,7 @@
     "libluv": "1.52.1-1",
     "libmakepkg-dropins": "20-2",
     "libmatio": "1.5.30-1",
-    "libmd": "1.2.0-1",
+    "libmd": "1.3.0-1",
     "libmm-glib": "1.24.2-1",
     "libmnl": "1.0.5-2",
     "libmodplug": "0.8.9.0-7",
@@ -325,7 +325,7 @@
     "libpgm": "5.3.128-4",
     "libpipeline": "1.5.8-1",
     "libpipewire": "1:1.6.9-1",
-    "libplacebo": "7.360.1-6",
+    "libplacebo": "7.360.1-7",
     "libplist": "2.7.0-3",
     "libpng": "1.6.59-1",
     "libportal": "0.11.0-1",
@@ -370,7 +370,7 @@
     "libusb": "1.0.30-1",
     "libusbmuxd": "2.1.1-2",
     "libutempter": "1.2.3-1",
-    "libutf8proc": "2.11.3-1",
+    "libutf8proc": "2.12.0-1",
     "libuv": "1.52.1-2",
     "libva": "2.24.1-1",
     "libvdpau": "1.5-4",
@@ -436,11 +436,11 @@
     "m4": "1.4.21-2",
     "make": "4.4.1-3",
     "man-db": "2.13.1-2",
-    "md4c": "0.5.3-1",
+    "md4c": "0.6.0-2",
     "mdadm": "4.6-2",
-    "mesa": "1:26.2.3-2",
+    "mesa": "1:26.2.4-1",
     "minizip": "1:1.3.2-3",
-    "mkinitcpio": "42.1-1",
+    "mkinitcpio": "42.2-1",
     "mkinitcpio-busybox": "1.36.1-1",
     "mobile-broadband-provider-info": "20251101-1",
     "mpdecimal": "4.0.1-3",
@@ -474,7 +474,7 @@
     "oniguruma": "6.9.10-1",
     "openal": "1.25.2-2",
     "opencore-amr": "0.1.6-2",
-    "openexr": "3.5.1-1",
+    "openexr": "3.5.2-1",
     "openh264": "2.6.0-2",
     "openjpeg2": "2.5.4-1",
     "openjph": "0.32.0-1",
@@ -543,12 +543,12 @@
     "sbc": "2.2-1",
     "sdbus-cpp": "2.3.1-1",
     "sddm": "0.21.0-7",
-    "sdl2-compat": "2.32.72-1",
-    "sdl3": "3.4.16-1",
+    "sdl2-compat": "2.32.74-1",
+    "sdl3": "3.4.18-1",
     "seatd": "0.9.3-1",
     "sed": "4.10-2",
     "serd": "0.32.10-1",
-    "shaderc": "2026.3-1",
+    "shaderc": "2026.4-1",
     "shadow": "4.20.0.arch1-1",
     "shared-mime-info": "2.5.1-2",
     "slang": "2.3.3-4",
@@ -561,7 +561,7 @@
     "spandsp": "0.0.6-7",
     "speex": "1.2.1-2",
     "speexdsp": "1.2.1-2",
-    "spirv-tools": "1:1.4.357.0-1",
+    "spirv-tools": "1:1.4.363.0-1",
     "sqlite": "3.53.4-1",
     "sratom": "0.6.22-1",
     "srt": "1.5.7-1",
@@ -610,10 +610,10 @@
     "vid.stab": "1.1.2-1",
     "vmaf": "3.2.1-1",
     "volume_key": "0.3.12-12",
-    "vulkan-icd-loader": "1.4.357.0-1",
-    "vulkan-mesa-implicit-layers": "1:26.2.3-2",
-    "vulkan-swrast": "1:26.2.3-2",
-    "vulkan-virtio": "1:26.2.3-2",
+    "vulkan-icd-loader": "1.4.363.0-1",
+    "vulkan-mesa-implicit-layers": "1:26.2.4-1",
+    "vulkan-swrast": "1:26.2.4-1",
+    "vulkan-virtio": "1:26.2.4-1",
     "wayland": "1.26.0-1",
     "wayland-protocols": "1.49-1",
     "webrtc-audio-processing-1": "1.3-5",
-- 
2.43.0

